### PR TITLE
part 1 -Cu 86b11cpnt together list UI

### PR DESCRIPTION
**Pull request by whamester**

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b11cpnt Together List UI](https://app.clickup.com/t/86b11cpnt)

**Comment by zeckkimon** [comment]
Task linked: [CU-86b11cm0w Home Page UI / Session UI](https://app.clickup.com/t/86b11cm0w)

**Review by ReinhardtBotha — APPROVED** [review]
Looks good!

**Review by zeckkimon — COMMENTED** [review]
Great structure! Thanks Wonnyo. Some comments below:

1. For Android, the header overlaps with the statusbar like this. But if it's ok with iOS, then that's all good.
image

2. For the Past Challenges, there should be an "All" filter. It wasn't revised in the mockups, but it's in the comments [here](https://www.figma.com/design/pmt5r2FAucfhLopa6dmEmr/Mockups?node-id=345-3312&m=dev)
image

File: src/services/challengeApi.tsx [hunk 1]
```diff
@@ -20,19 +21,24 @@ export const getOngoingChallenges = async (
...
-  return response.data;
+  return data.data as Array<ChallengeResponseType>;
 };
 
 export const getPastChallenges = async (
-  filterStatus: ChallengeStatusEnum | undefined,
+  filterStatus: `${ChallengeStatusEnum}` | undefined,
   sortDesc: number,
 ) => {
   const status = filterStatus ? `&filterStatus=${filterStatus}` : ``;
```
Comment (zeckkimon) [inline src/services/challengeApi.tsx:34]: This should be:

`  const status = filterStatus ? filterStatus : ``;`

File: src/components/listItems/ChallengeCard.tsx [hunk 2]
```diff
@@ -5,61 +5,98 @@ import { Text } from "@src/components/ui/typography";
...
-  const progressBarLabel = (
-    <Text>
-      {progress}/{goalPoints}
-    </Text>
-  );
+  const progress =
+    challenge.members.reduce(
+      (accu: any, curr: any) => accu + safenumber(curr.points),
+      0,
+    ) || 100;
```
Comment (zeckkimon) [inline src/components/listItems/ChallengeCard.tsx]: Why is there `|| 100 `?

**Review by Victorportus — APPROVED** [review]
Same problem as Beck. On Android needs some padding on the top, but for IOS looks good

File: src/services/challengeApi.tsx (hunk 1 above)
Comment (whamester) [reply src/services/challengeApi.tsx:34]: why?

File: src/components/listItems/ChallengeCard.tsx (hunk 2 above)
Comment (whamester) [reply src/components/listItems/ChallengeCard.tsx]: removed

**Review by zeckkimon — APPROVED** [review]

**Pull request closed without merging**